incubator/couchdb/templates/statefulset.yaml

```diff
@@ -62,10 +71,11 @@ spec:
           resources:
 {{ toYaml .Values.resources | indent 12 }}
           volumeMounts:
+          - name: config-storage
+            mountPath: /opt/couchdb/etc/default.d
+            subMountPath: chart.ini
```

Comment:
@mainephd I can't find `subMountPath` documented anywhere -- is this an alias for `subPath`? I also found that this updated chart fails to validate on my older Helm 2.7.2 with

> Error: error validating "": error validating data: found invalid field subMountPath for v1.VolumeMount

When I upgraded to 2.9.1 I was able to validate and then install it successfully. I'm wondering if was can improve the backwards compatibility here?

Reply:
@kocolosk it should be `subPath` at least as verified on my k8s cluster running 1.9.7